spec/lib/pdf_filler/md_el101_pdf_spec.rb

```diff
@@ -32,8 +78,25 @@
         allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_46).and_return 300
       end
 
-      it 'outputs the amount to be refunded' do
+      it "fills out the required fields" do
+        expect(pdf_fields["First Name"]).to eq("Mary")
+        expect(pdf_fields["Primary MI"]).to eq("A")
+        expect(pdf_fields["Last Name"]).to eq("Lando")
+        expect(pdf_fields["SSNTaxpayer Identification Number"]).to eq("555123666")
         expect(pdf_fields["2 Amount of overpayment to be refunded to you                                         2"]).to eq "300"
+        expect(pdf_fields["3"]).to eq("0")
+        expect(pdf_fields["ERO firm name"]).to eq "FileYourStateTaxes"
+        expect(pdf_fields["to enter or generate my PIN"]).to eq "23456"
+        expect(pdf_fields["Primary signature"]).to eq "Mary Lando"
+        expect(pdf_fields["Date"]).to eq Date.today.strftime("%F")
+        expect(pdf_fields["Spouses First Name"]).to eq("")
+        expect(pdf_fields["Spouse MI"]).to eq("")
+        expect(pdf_fields["Spouses Last Name"]).to eq("")
+        expect(pdf_fields["SSNTaxpayer Identification Number_2"]).to eq("")
+        expect(pdf_fields["ERO firm name_2"]).to eq ""
+        expect(pdf_fields["to enter or generate my PIN_2"]).to eq ""
+        expect(pdf_fields["Spouses signature"]).to eq ""
+        expect(pdf_fields["Date_2"]).to eq ""
```

Comment:
[dust] some of these fields need to be filled out regardless of taxes being owed or refunded. should there be like a single filer block for the common fields that are filled out without spouse fields being filled out?

Reply:
ah yes